### PR TITLE
Adds bellsoft resource

### PR DESCRIPTION
**Pull request by pivotal-david-osullivan**

### CODE REVIEW

File: resources/bellsoft-liberica/bellsoft.go [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+		Version:        b.Source.Version,
+		Header:			b.Source.Token,
+	}
+
+	if err := md.load(); err != nil {
+		return in.Result{}, err
+	}
+	
+	uri := md.versions[b.Version]
+	url, err := url.Parse(uri)
```
Comment (anthonydahanne) [inline resources/bellsoft-liberica/bellsoft.go:74]: url clashes with import

```suggestion
	downloadUrl, err := url.Parse(uri)
```

File: resources/bellsoft-liberica/bellsoft.go [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+	if err != nil {
+		return in.Result{}, err
+	}
+
+	sha256, err := in.Artifact{
+		Name:        b.name(url.Path),
```
Comment (anthonydahanne) [inline resources/bellsoft-liberica/bellsoft.go:80]:
```suggestion
		Name:        b.name(downloadUrl.Path),
```

**Review by anthonydahanne — CHANGES_REQUESTED** [review]
no authentication management?

**Comment by pivotal-david-osullivan** [comment]
'Token' & 'Header' fields have been added the Bellsoft/Metadata structs to support optional headers/auth

**Comment by anthonydahanne** [comment]
> 'Token' & 'Header' fields have been added the Bellsoft/Metadata structs to support optional headers/auth

oh I see now, the substring thing on the header. Thanks!

**Review by anthonydahanne — APPROVED** [review]

**Pull request merged**